fidesctl/src/fidesctl/core/apply.py

```diff
@@ -64,7 +64,7 @@ def echo_results(action: str, resource_type: str, resource_count: int) -> None:
     """
     Echo out the results of the apply.
     """
-    echo_green(f"{action.upper()} {resource_count} {resource_type} resources.")
+    echo_green(f"{action.upper()} {resource_count} {resource_type} resource(s).")
```

Comment:
I forgot we have this helper here, so I think it would be pretty easy to dynamically pluralize the text:
```suggestion
	pluralize = "s" if resource_count != 1 else ""
    echo_green(f"{action.upper()} {resource_count} {resource_type} resource{pluralize}.")
```

Reply:
This just feels like too much....`(s)` is a well understood shorthand for this exact situation